CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 lib_logging change log
 ======================
 
+UNRELEASED
+----------
+
+  * UPDATED: Fixed type conversion warnings
```

Comment:
Following our normal convention this should be FIXED:   Type conversion warnings